src/tools/miri/src/shims/time.rs

```diff
@@ -45,6 +45,9 @@ pub trait EvalContextExt<'mir, 'tcx: 'mir>: crate::MiriInterpCxExt<'mir, 'tcx> {
                 relative_clocks = vec![
                     this.eval_libc_i32("CLOCK_MONOTONIC")?,
                     this.eval_libc_i32("CLOCK_MONOTONIC_COARSE")?,
+                    // This is the equivalent to `CLOCK_UPTIME_RAW` in the macos section
+                    // We support it, because std relies on it
```

Comment:
```suggestion
                    // Unlike `CLOCK_MONOTONIC`, this *is* supposed to increment during system sleep...
                    // but that's not really something a program running inside Miri can tell, anyway.
                    // We support it, because std relies on it
```